frontend/src/App.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+
```

Comment:
Maybe group the imports together? Remove the extra white spaces between imports.